LayoutTests/imported/w3c/web-platform-tests/css/css-grid/grid-lanes/tentative/subgrid/track-sizing/grid-lanes-subgrid-flex-ref.html

```diff
@@ -59,19 +59,19 @@
 </grid>
 
 <grid>
-  <item style="width: 4ch">1</item>
+  <item>1</item>
   <subgrid>
     <item>2</item>
     <item>3</item>
-    <item>4</item>
+    <item style="width: 4ch">4</item>
```

Comment:
This is a little hard to follow in the source, maybe you can tag this onto item 2 also?

Reply:
Sounds good